vms/evm.ts

```diff
@@ -168,6 +189,16 @@ export default class EVM {
         return { txHash, rawTransaction };
     }
 
+    async getGasPrice(): Promise<number> {
+        return await this.web3.eth.getGasPrice()
```

Comment:
since the function is returning a Promise, we can drop the await in the return statement and pass through the Promise directly